.github/workflows/main.yml

```diff
@@ -110,3 +110,21 @@ jobs:
         with:
           name: collection
           path: theforeman-operations-*.tar.gz
+
+  molecule:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: gofrolist/molecule-action@v2
+        with:
+          molecule_working_dir: roles/foreman_installer
```

Comment:
in `forklift`, I just used a `for` loop over all roles that have a molecule folder, and no action:
https://github.com/theforeman/forklift/blob/master/.github/workflows/main.yml#L32-L38

is this action buying us anything? it's not official…